src/dev/ce-migration/index.md

```diff
@@ -3,5 +3,115 @@
 
-## Migration guide to 2023.2
+## Migration guide from any previous version to 2023.2
 
+Besides various fixes, enhancements and performance improvements the most notable change recently introduced is the concept of workspaces. For Community Edition it means there is a **common shared workspace (global workspace)** for all users where all projects are stored, instead of having a personal or organizational namespace for projects.
+
+:::tip
+In case you do not need your previous data, we advise to start with clean deployment without the need to follow this migration guide.
+:::
+
+**Upgrading to 2023.2**
+
+ 1. Sync all your projects. Hit synchronise on all devices that have local changes (mobiles, QGIS plugins)
+
+ 2. Remove downloaded projects from all devices (mobiles, QGIS plugins)
+
+ 3. Backup your database ⚠️
+
+```bash
+$ docker exec mergin-db pg_dump -U postgres -Fc postgres > pg_backup.dump
+```
+
+ 4. Stop all running Mergin Maps services (from project root folder)
+```bash
+$ docker-compose -f docker-compose.yml stop
+```
+
+ 5. Pull the latest changes
+```bash
+$ git pull
+```
+
+ 6. Amend environment variables (.env file). **Important** ⚠️
+
+As mentioned earlier, CE operates with one global workspace. We will set it up now.
+Specify its name with the following environment variable:
+
+ - `GLOBAL_WORKSPACE=ShinyWorkspace` - name of your workspace. A good fit is a name of your company or team. This value *should not be changed* later.
+
+:::tip
+You can find all available environment variables in "TODO:add file here" together with a tutorial how to set them up.
+:::
+
+Further, you need to set a default role for people in your workspace. **Pick one** of these options
+(*learn more about our [permissions and roles system here](../../manage/permissions)*):
+
+ - `GLOBAL_READ=0` everyone will have guest role (without access to any project unless explicitly granted)
+ - `GLOBAL_READ=1` everyone will have reader role (they can read/download all projects in the workspace)
+ - `GLOBAL_WRITE=1` everyone will have writer role (they can contribute to all projects in the workspace, e.g. upload files)
+ - `GLOBAL_ADMIN=1` everyone will have admin role (they can create new projects and share projects with others)
+
+You can specify the maximum storage for your shiny new workspace 🌟 with the following variable:
+
+ - `GLOBAL_STORAGE=1048576` - workspace storage in MB
+
+:::tip
+New users can be created from Mergin Maps administration panel, by navigating to `<your_url>/admin`.
+:::
+
+ 7. Make sure projects volume mounts in `docker-compose` file still match. Switch to new server version and postgres to at least version 12 (14 recommended) by running new docker containers:
+```bash
+$ docker-compose -f docker-compose.yml up
+```
+
+ 8. Restore backup from older postgres version, e.g.:
+
+```bash
+$ docker cp pg_backup.dump merginmaps-db:/tmp
```

Comment:
@tomasMizera could you double check container names are those ones from docker-compose file (just to avoid confusion)?

Reply:
Looks OK